module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Find-EntraPermission.md

```diff
@@ -85,7 +77,7 @@ be74164b-cff1-491c-8741-e671cb536e13 Admin   Policy.ReadWrite.ApplicationConfigu
 
 ###Example 2. Get a list of permissions for the Read permissions of the Application domain.
 ```powershell
-PS C:\>Find-EntraPermissions application.Read | Format-List
```

Comment:
Kindly remove the command line prompts e.g. PS C:\>.

Check how Get-EntraUser and other commands look like.